src/TinyLittleMvvm/IWindowManager.cs

```diff
@@ -23,5 +25,23 @@ public interface IWindowManager {
         /// <param name="scope">Optional IoC scope for the window.</param>
         /// <returns>The window.</returns>
         Window ShowWindow(object viewModel, Window owningWindow = null, IServiceScope scope = null);
+
+        /// <summary>
+        /// Shows a window for a given view model type as a Dialog.
+        /// </summary>
+        /// <param name="owningWindow">An optional owner for the new window.</param>
+        /// <param name="scope">Optional IoC scope for the window.</param>
+        /// <typeparam name="TViewModel">The type of the view model.</typeparam>
+        /// <returns>The dialog window.</returns>
+        Window ShowDialog<TViewModel>(Window owningWindow = null, IServiceScope scope = null);
```

Comment:
IMHO it doesn't make sense to return the window, because it's already closed when this method returns. I suggest that returning the view model is more appropriate.